build.sbt

```diff
@@ -225,7 +247,12 @@ lazy val pythonSettings = Seq(
       baseEnv :+ "ARROW_PRE_0_15_IPC_FORMAT" -> "1"
     }
     val ret = Process(
-      Seq("pytest") ++ args,
+      Seq(
+        "/bin/bash",
+        "-c",
+        "source $(conda info --base)/etc/profile.d/conda.sh &&" +
```

Comment:
Is this pretty standard?

Reply:
This is the line added to `.bashrc`/`.bash_profile` when installing conda in non-batch mode (without `-b`) for initialization purposes; without this, we can't use commands such as `conda activate`.